<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nbaTeams</class>
 <widget class="QWidget" name="nbaTeams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="dataView">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>151</y>
     <width>621</width>
     <height>281</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.14286pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.14286pt; font-weight:600;&quot;&gt;Eastern Division&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="EasternComboBox">
    <item>
     <property name="text">
      <string>* Choose Team From Eastern Division*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>All Eastern Division Teams</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brooklyn Nets</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Boston Celtics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New York Knicks</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Philadelphia 76ers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Toronto Raptors</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chicago Bulls</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cleveland Cavaliers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Detroit Pistons</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Indiana Pacers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Milwaukee Bucks</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Atlanta Hawks</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Charlotte Hornets</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Miami Heat</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Orlando Magic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Washington Wizards</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.14286pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.14286pt; font-weight:600;&quot;&gt;Western Division&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="westernComboBox">
    <item>
     <property name="text">
      <string>* Choose Team From Western Division *</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>All Western Division Teams</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Denver Nuggets</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Minnesota Timberwolves</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oklahoma City Thunder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Portland Trail Blazers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Utah Jazz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Golden State Warriors</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Los Angeles Clippers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Los Angeles Lakers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Phoenix Suns</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sacramento Kings</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dallas Mavericks</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Houston Rockets</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Memphis Grizzlies</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Orleans Pelicans</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>San Antonio Spurs</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
